packages/syft/src/syft/client/client.py

```diff
@@ -529,24 +529,36 @@ def create_project(
         return project
 
     def sync_code_from_request(self, request):
-        code = request.code
+        # relative
+        from ..service.code.user_code import UserCode
+        from ..store.linked_obj import LinkedObject
+
+        code: Union[UserCode, SyftError] = request.code
         if isinstance(code, SyftError):
             return code
-        elif code is None:
-            return SyftError(message="no code inside request")
 
         code = deepcopy(code)
+        code.node_uid = self.id
```

Comment:
node_uid and user_verify_key should be updated to current node and credentials, otherwise it reflects the same of the low-side node.